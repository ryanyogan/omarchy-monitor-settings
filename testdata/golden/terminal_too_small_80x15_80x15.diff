Visual regression detected!

=== EXPECTED ===
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                               [31mTerminal too small[0m                               
                        [31mPlease resize to at least 80x20[0m                         
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                

=== ACTUAL ===
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                               Terminal too small                               
                        Please resize to at least 80x20                         
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                

=== DIFF INFO ===
Expected length: 1232
Actual length: 1214
Expected hash: 0ccc25d5bbbe18b9ae412ea8c9311e562a257ed7613d7429025c5084b76c92e8
Actual hash: 68a87ee520f13c184888b3434d0294c5d9a964504785a6ad21f59a4c7848d09a
