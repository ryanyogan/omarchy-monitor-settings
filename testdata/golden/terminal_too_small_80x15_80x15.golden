# Visual Golden File
# Name: terminal_too_small_80x15
# Dimensions: 80x15
# Hash: 0ccc25d5bbbe18b9ae412ea8c9311e562a257ed7613d7429025c5084b76c92e8

                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                               [31mTerminal too small[0m                               
                        [31mPlease resize to at least 80x20[0m                         
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                